router/src/main/java/io/micronaut/web/router/AnnotatedFilterRouteBuilder.java

```diff
@@ -41,27 +41,23 @@
 @Singleton
 public class AnnotatedFilterRouteBuilder extends DefaultRouteBuilder implements BeanDefinitionProcessor<Filter> {
 
-    private final BeanContext beanContext;
     private final ServerContextPathProvider contextPathProvider;
 
     /**
      * Constructor.
      *
-     * @param beanContext The bean context
      * @param executionHandleLocator The execution handler locator
      * @param uriNamingStrategy The URI naming strategy
      * @param conversionService The conversion service
      * @param contextPathProvider The server context path provider
      */
     @Inject
     public AnnotatedFilterRouteBuilder(
-            BeanContext beanContext,
```

Comment:
This is a breaking change and thus must be reverted

Reply:
Argument restored.